### PR TITLE
fix: put actual user status on omnichannel queue view

### PR DESCRIPTION
**Pull request by KevLehman**
OC-203

### CODE REVIEW

**Review by murtaza98 — APPROVED** [review]

File: apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx [hunk 1]
```diff
@@ -88,6 +88,25 @@ const QueueListTable = (): ReactElement => {
 		return query;
 	}, [sortBy, sortDirection, itemsPerPage, current, filters.status, filters.departmentId, filters.servedBy]);
 
+	const getUserStatus = (status?: string) => {
```
Comment (murtaza98) [inline apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx:91]:
```suggestion
	const getUserStatus = (status?: UserStatus) => {
```

File: apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx [hunk 2]
```diff
@@ -88,6 +88,25 @@ const QueueListTable = (): ReactElement => {
...
+		if (!status) {
+			return t('Offline');
+		}
+
+		switch (status) {
+			case 'online':
```
Comment (murtaza98) [inline apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx:97]:
```suggestion
			case UserStatus.ONLINE:
```

File: apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx [hunk 3]
```diff
@@ -88,6 +88,25 @@ const QueueListTable = (): ReactElement => {
...
+				return t('Online');
+			case 'away':
```
Comment (murtaza98) [inline apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx:99]:
```suggestion
			case UserStatus.AWAY:
```

File: apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx [hunk 4]
```diff
@@ -88,6 +88,25 @@ const QueueListTable = (): ReactElement => {
...
+				return t('Away');
+			case 'busy':
```
Comment (murtaza98) [inline apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx:101]:
```suggestion
			case UserStatus.BUSY:
```

File: apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx [hunk 5]
```diff
@@ -88,6 +88,25 @@ const QueueListTable = (): ReactElement => {
...
+				return t('Busy');
+			case 'offline':
```
Comment (murtaza98) [inline apps/meteor/client/views/omnichannel/queueList/QueueListTable.tsx:103]:
```suggestion
			case UserStatus.OFFLINE:
```

**Review by murtaza98 — CHANGES_REQUESTED** [review]
A small suggestion to use enums

**Review by murtaza98 — APPROVED** [review]

**Review by murtaza98 — APPROVED** [review]

**Review by aleksandernsilva — APPROVED** [review]

**Pull request merged**